source/filesystem.hpp

```diff
@@ -23,7 +23,7 @@ namespace reshade::filesystem
 	public:
 		path() { }
 		path(const char *data) : _data(data) { }
-		path(const std::string &data) : _data(data) { }
+		path(std::string data) : _data(std::move(data)) { }
```

Comment:
Ah, right, I only saw the pass-by-value, but didn't realize that works out with move semantics anyway.